### ISSUE TITLE
Allow multiple status lists in SD-JWT VC

### ISSUE THREAD

**Issue by bj-ms** [oauth-wg/draft-ietf-oauth-status-list#151]
In our current use case, we have implemented the bitstring status list (https://www.w3.org/TR/vc-bitstring-status-list/) and want to keep it as compatible as possible.

One of the incompatibilities we discovered in chapters 6.2, 6.3, and 6.4 is that the status list property is limited to a status list.
We would suggest allowing a list of status lists in those chapters instead.

**Comment by bj-ms** [comment]
No, it was more about compatibility but as you say you take another approach.

Are there some differences in privacy concerns between IETF Status List and Bitstring that you know of?

**Comment by paulbastian** [comment]
@bj-ms Are there any open questions on this one or do you have further feedback?

**Comment by bj-ms** [comment]
nothing more to say, I will close the issue
